packages/fast-usdc/src/exos/settler.js

```diff
@@ -75,19 +108,38 @@ export const prepareSettler = (zone, { statusManager }) => {
           );
         }
 
-        // TODO disperse funds
-        // ~1. fee to contractFeeAccount
-        // ~2. remainder in poolAccount
+        // Disperse funds
+
+        const { repayer, settlementAccount } = this.state;
+        const received = AmountMath.make(USDC, amountInt);
+        const { zcfSeat: settlingSeat } = zcf.makeEmptySeatKit();
+        const { calculateSplit } = makeFeeTools(feeConfig);
+        const split = calculateSplit(received);
```

Comment:
> // TODO discern between SETTLED and OBSERVED; each has different fees/destinations

We need to check if the TX we are going to settle is `Advanced` or `Observed`. If `Observed`, we shouldn't charge fees because there was no advance. I suspect we also need to `.transfer()` the full `amountInt` to the EUD in `aux.recipientAddress`.

`StatusManager`'s interfaces might require an update to accommodate this.